packages/ui/src/views/pull-request/pull-request-list.tsx

```diff
@@ -30,61 +30,65 @@ export function PullRequestList({ pullRequests, open_prs, closed_prs }: PullRequ
     })
   }, [headerFilter])
 
+  const onOpenClick = () => {
+    setHeaderFilter('open')
+  }
+  const onCloseClick = () => {
+    setHeaderFilter('closed')
+  }
+  if (!filteredData?.length) return null
   return (
-    <>
-      {!!filteredData?.length && (
-        <StackedList.Root>
-          <StackedList.Item isHeader disableHover>
-            <StackedList.Field
-              title={
-                <PullRequestListHeaderTitle
-                  headerFilter={headerFilter}
-                  setHeaderFilter={setHeaderFilter}
-                  open_prs={open_prs}
-                  closed_prs={closed_prs}
-                />
-              }
+    <StackedList.Root>
+      <StackedList.Item isHeader disableHover>
+        <StackedList.Field
+          title={
+            <PullRequestListHeaderTitle
+              headerFilter={headerFilter}
+              onOpenClick={onOpenClick}
+              onCloseClick={onCloseClick}
+              openPRs={openPRs}
+              closedPRs={closedPRs}
             />
+          }
+        />
+      </StackedList.Item>
+      {filteredData?.map((pullRequest, pullRequest_idx) => (
```

Comment:
Same here, you know this definitely exists
```suggestion
      {filteredData.map((pullRequest, pullRequest_idx) => (
```